2-api/key-concepts.md

```diff
@@ -2,36 +2,42 @@
 
 # Key concepts
 
-To best understand the reference documentation, you must first know these key concepts.
+To best understand the reference documentation, you must first understand these key concepts.
 
 ## Attribute
 
-A comment label used for review. The most popular attributes are the toxicity and severe toxicity models.
+An **attribute** is a comment label used for review. The most popular
+attributes are the toxicity and severe toxicity models.
```

Comment:
Could we emphasize TOXICITY and SEVERE_TOXICITY in this sentence, since they they are the name of attributes?
Might be helpful elsewhere in the doc, too, but lemme know what you think.
The style we use for naming attributes seems to be to capitalize them, so perhaps that is the right choice of emphasis here, rather than boldface, initial caps, italics, etc.